DFC.App.MatchSkills.Application/Session/Services/SessionService.cs

```diff
@@ -26,10 +31,22 @@ public SessionService(ICosmosService cosmosService, IOptions<SessionSettings> se
             _sessionSettings = sessionSettings;
         }
 
-        public async Task<string> CreateUserSession(string previousPage, string currentPage)
+        public async Task<string> CreateUserSession(string previousPage, string currentPage, string sessionIdFromCookie = null)
```

Comment:
Any reason this is Async but doesn't await any tasks?

Reply:
It does. Use eyes pls.